.travis.yml

```diff
@@ -0,0 +1,8 @@
+sudo: false
```

Comment:
Don't you need `sudo` for `apt-get install`?

Reply:
Good question, I think this line is not required. I made several tests on my own fork before and forgot to test without this line.